src/quacc/recipes/emt/md.py

```diff
@@ -0,0 +1,123 @@
+"""
+Molecular Dynamics recipes for EMT.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase.calculators.emt import EMT
+from ase.md.velocitydistribution import (
+    MaxwellBoltzmannDistribution,
+    Stationary,
+    ZeroRotation,
+)
+
+from quacc import job
+from quacc.runners.ase import Runner
+from quacc.schemas.ase import summarize_md_run
+from quacc.utils.dicts import recursive_dict_merge
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from ase.atoms import Atoms
+
+    from quacc.schemas._aliases.ase import DynSchema
+
+
+@job
+def md_job(
+    atoms: Atoms,
+    maxwell_boltzmann_params: dict[str, Any] | None = None,
+    md_params: dict[str, Any] | None = None,
+    **calc_kwargs,
+) -> DynSchema:
+    """
+    Carry out a Molecular Dynamics calculation.
+
+    !!! Note "Units"
+
+        Quacc does not follow ASE standards for Molecular Dynamics units.
+
+        Quacc ALWAYS uses the following units:
+
+        - Time: femtoseconds (fs)
+        - Pressure: GPa
+        - Temperature: Kelvin (K)
+        - Compressibility: 1/GPa
+
+    !!! Note "Keywords"
+
+        Additionally, Quacc uses the keywords `fix_com` and `fix_rot` instead of
+        `fixcm` and `fixrot` to fix the center of mass and rotation, respectively.
```

Comment:
If this is just a matter of renaming and nothing else, let's just stick with the ASE ones. If you approve, let me know and I can revert the change here.

Reply:
Yes sure!